package.json

```diff
@@ -112,6 +113,9 @@
           "useTabs": false
         }
       ]
+    },
+    "globals": {
+      "localStorage": true
```

Comment:
Potentially better to add `/* global localStorage */` to the top of `src/index.web.ts`, since it's only available as a global on the web platform, and not to the rest of the code...

ref: https://eslint.org/docs/rules/no-undef